src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -4834,7 +4832,6 @@ private bool IsUsingAliasInScope(string name)
                 return BadExpression(node, boundLeft);
             }
 
-            // PROTOTYPE(default) unify with case above
             // No member accesses on default
             if (boundLeft.IsLiteralDefault())
```

Comment:
I addressed your other feedback in latest commit, but I couldn't quite see how to factor as you suggested here.